src/taskpane/components/Excel.App.tsx

```diff
@@ -72,13 +73,17 @@ export default class App extends React.Component<AppProps, AppState> {
 
     if (!isOfficeInitialized) {
       return (
-        <Progress title={title} logo="assets/logo-filled.png" message="Please sideload your addin to see app body." />
+        <Progress
+          title={title}
+          logo={require("./../../../assets/logo-filled.png")}
```

Comment:
Why are we requiring to put the full path files now?

Reply:
It's more then just the path . . . it's the need for the "require".  Currently the configuration for webpack has a mix of copying the images as files in a specific directory with original name and having them handled by webpack with hashed names and each of these files get's used in different places.  Part of this change is to standardize on just the automatic webpack handling of the images which means some of the references need to be updated in order for webpack to recognize them as needing to be managed and updated to the hashed image name.

Reply:
You shouldn't need to use require here. The relative path should be enough.

Reply:
If I don't, then the images references in the code output by webpack aren't updated to the image names and path created by webpack and they aren't loaded.  The reason it was working before was because of the second set of images that were being copied as files which is what I was removing (don't need two sets).

There are variation of this type of thing listed on the web (doing the require as an import statement or using a custom .d.ts file) the the effect is basically the same.